src/app/models/settings.ts

```diff
@@ -9,6 +9,8 @@ export interface AppSettings {
   oscReceivingPort: number;
   enableDesktopNotifications: boolean;
   enableXSOverlayNotifications: boolean;
+  exitInSystemTray: boolean;
+  startInSystemTray: boolean;
 }
```

Comment:
So these interfaces have a `version` property that gets bumped whenever they are modified. This is so that people's existing configurations can be migrated once they update. 

In this case, two new fields got added, which means the `version` property should be bumped from `4` to `5`, and a migration should be written for it in `src/app/migrations/app-settings.migrations.ts`. If you take a look at the previous migrations in there, you'll see it's quite simple!